obspy/io/sh/tests/test_evt.py

```diff
@@ -91,8 +91,9 @@ def test_populate_waveform_id(self):
         # read invenroty - FUR with channels, WET without channels
         inv = read_inventory().select(channel='HH?')
         inv[0][1].channels = []
-        picks = read_events(fname, inventory=inv, id_default='BLA.{}.11.DH{}',
-                            id_map={'UBR': 'BLB.{}.00.BH{}'})[0].picks
+        cat = read_events(fname, inventory=inv, id_default='BLA.{}.11.DH{}',
+                          id_map={'UBR': 'BLB.{}.00.BH{}'}, warn=False)
```

Comment:
here passing `id_map` and not `seedid_map` is voluntary?

Reply:
Thanks for spotting this. The use of `id_map` and `id_default` are non-intentional here. They are only supported for backwards compatibility.